providers/infrastructure-aws/v2.0.2/cconly/overlay-kube-apiserver-admission.yaml

```
@@ -0,0 +1,192 @@
+        type: object
+        default: {}
+        properties:
+          deactivated:
+            type: boolean
+            description: "deactivated disables the patches for Pod Security Standard via AdmissionConfiguration."
+          enforce:
+            type: string
+            enum: ["", "privileged", "baseline", "restricted"]
+            description: "enforce sets the level for the enforce PodSecurityConfiguration mode. One of \"\", privileged, baseline, restricted."
+            nullable: true
+          enforceVersion:
+            type: string
+            #! Version defaults to v1.24 for now. When v1.25 is part of tanzu-framework, the defaulting should be done
+            #! by a defaulting webhook instead for extended logic.
+            default: "v1.24"
+            description: "enforceVersion sets the version for the enforce PodSecurityConfiguration mode."
+          audit:
+            type: string
+            enum: ["", "privileged", "baseline", "restricted"]
+            description: "audit sets the level for the audit PodSecurityConfiguration mode. One of \"\", privileged, baseline, restricted."
+            nullable: true
+          auditVersion:
+            type: string
+            #! Version defaults to v1.24 for now. When v1.25 is part of tanzu-framework, the defaulting should be done
+            #! by a defaulting webhook instead for extended logic.
+            default: "v1.24"
+            description: "auditVersion sets the version for the audit PodSecurityConfiguration mode."
+          warn:
+            type: string
+            enum: ["", "privileged", "baseline", "restricted"]
+            description: "warn sets the level for the warn PodSecurityConfiguration mode. One of \"\", privileged, baseline, restricted."
+            nullable: true
+          warnVersion:
+            type: string
+            #! Version defaults to v1.24 for now. When v1.25 is part of tanzu-framework, the defaulting should be done
+            #! by a defaulting webhook instead for extended logic.
+            default: "v1.24"
+            description: "warnVersion sets the version for the warn PodSecurityConfiguration mode."
+          exemptions:
+            type: object
+            description: "exemption configuration for the PodSecurityConfiguration."
+            properties:
+              namespaces:
+                type: array
+                items:
+                  type: string
+                description: "namespaces excluded to apply PodSecurityConfiguration Admission."
+  #@overlay/remove
+  #@overlay/match by=overlay.map_key("name"),missing_ok=True
+  - name: eventRateLimitConf
+  #@overlay/append
+  - name: eventRateLimitConf
+    required: false
+    schema:
+      openAPIV3Schema:
+        type: string
+        default: ""
+  #@overlay/match missing_ok=True
+  patches:
+  #@overlay/remove
+  #@overlay/match by=overlay.map_key("name"),missing_ok=True
+  - name: admissionConfiguration
```

```suggestion
  - name: podSecurityStandard
```
This needs to keep on removing the upstream patch for `podSecurityStandard`